why3gospel/why3gospel.ml

```diff
@@ -160,12 +155,14 @@ let read_channel env path file c =
   (* let f = filter_equiv f in
    * if f <> [] then print_equiv file f; *)
   if Debug.test_flag print_modules then begin
+    let open Why3.Ident in
```

Comment:
I am OK with the `let open` syntax, but I believe this is not the most appreciated style by everyone here, especially core Why3 developers :) 
@belolourenco @backtracking, can you add your thoughts here?

Reply:
The goal was to reduce the number of globally open modules since I find it harder to understand what comes from where, but of course I'm happy to revert this if needed.